requirements.in

```diff
@@ -13,7 +13,6 @@
 # In order to create the requirements.txt file, execute
 # pip-compile --output-file requirements.txt requirements.in (pip-tools package is needed).
 
-PyYAML<3.13
```

Comment:
this change isn't enough - you have to regenerate the `requirements.txt` file (using `pip-compile` - read comment on `requirements.in`).
when you do so, note the comments on the top of the current `requirements.txt` file - there's some manual modifications that must be made as well.